css/discussions.css

```diff
@@ -0,0 +1,14 @@
+/* Discussions CSS */
+
+.replies {
+	padding-left:7px;
+	border-left:solid #cccccc 16px;
+}
+
+.discussion-read-state-btn {
+	top:32px;
+}
+
+div.entry-content {
+	padding-left:0px !important;
```

Comment:
Interesting choice. It appears that the threaded version of the discussion continues to indent replies to sub-replies at 30px per:
![screen shot 2016-08-24 at 9 43 41 am](https://cloud.githubusercontent.com/assets/8975317/17933243/5caf36d0-69e1-11e6-9915-02b878b67585.png)


Whereas the non-threaded discussion type indents everything following the first of a reply to the original post (that is, the first level of sub-replies):
![screen shot 2016-08-24 at 9 42 51 am](https://cloud.githubusercontent.com/assets/8975317/17933239/564ed674-69e1-11e6-949e-8aafe3106afc.png)


Their change would blur the distinction between a first-level/direct reply and the second-level/sub-reply:
![screen shot 2016-08-24 at 9 49 53 am](https://cloud.githubusercontent.com/assets/8975317/17933252/695d2360-69e1-11e6-9380-441ca11acbd7.png)


If they're attempting to be egalitarian, they might want to blow the sub-reply icons back up to full size too. But if they're just doing this to avoid extreme sub-reply nesting, perhaps turning off threaded replies for discussions is all they need to do. In any case, this seems to limit what instructors may want to do with their discussions.